.python-version

```diff
@@ -1 +1 @@
-3.6.5
+3.7.0
```

Comment:
3.7.5?

Reply:
Circleci pyenv does not support 3.7.5, so this has to be 3.7.0 for now, as @milesgranger pointed out earlier.